resources/service_guides.json

```diff
@@ -1,6 +1,10 @@
 [
   {
-    "name": "VAT End-to-End Service Guide",
+    "name": "VAT (MTD) end-to-end service guide",
     "context": "/guides/vat-mtd-end-to-end-service-guide"
```

Comment:
Similarly, is it not the case that the previously existing one should be removed?